tiledb/sm/array_schema/array_schema.cc

```diff
@@ -1128,6 +1132,34 @@ const std::string& ArraySchema::get_enumeration_path_name(
   return iter->second;
 }
 
+void ArraySchema::load_enumeration(Context& ctx, const std::string& enmr_name) {
```

Comment:
This function needs to support `remote_array` case like the `Array::load_enumerations`. I think we can reuse the same route and only minor adjustments will be needed in serialization functions to support this.